javalib/src/main/scala/java/io/File.scala

```diff
@@ -541,4 +571,22 @@ object File {
                                minLength = true,
                                throwOnError = true)
 
+  // Ported from Apache Harmony
+  private def checkURI(uri: URI): Unit = {
+    def throwExc(msg: String): Unit =
+      throw new IllegalArgumentException(s"$msg: $uri")
+    def compMsg(comp: String): String = s"Found $comp component in URI"
```

Comment:
Please put a newline after the `=` here.

Reply:
I also added a blank line before the body of the method.